Assembler.UnitTests/AssemblerTests.cs

```diff
@@ -0,0 +1,165 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using Assembler.Core;
+using Assembler.Core.Entities;
+using Assembler.Core.Enums;
+using Moq;
+using NUnit.Framework;
+
+namespace Assembler.UnitTests
+{
+    [TestFixture]
+    public class MessagesAssemblerTests
+    {
+        private Mock<IResolver<FrameType, IHandler<BaseFrame, BaseMessageInAssembly>>> _resolverMock;
+        private Mock<IConverter<BaseMessageInAssembly, BaseAssembledMessage>> _converterMock;
+        private Mock<ITimeBasedCache<BaseMessageInAssembly>> _cacheMock;
+        private Mock<IHandler<BaseFrame, BaseMessageInAssembly>> _firstHandlerMock;
+        private Mock<IHandler<BaseFrame, BaseMessageInAssembly>> _secondHandlerMock;
+        private Base.MessagesAssembler _assembler;
+
+        private List<BaseAssembledMessage> _assembledMessages;
+        private List<Mock<IHandler<BaseFrame, BaseMessageInAssembly>>> _handlersList;
```

Comment:
Not critical, but maybe change this to `_handlerMocks`

Reply:
f

Reply:
👍